src/backends/smtlib2.rs

```diff
@@ -14,6 +14,7 @@ use petgraph::EdgeDirection;
 
 use backends::backend::{Logic, SMTBackend, SMTError, SMTNode, SMTResult};
 use theories::{bitvec, core, integer};
+use super::backend::SMTRes;
```

Comment:
Should this instead be `use backends::backend::{.., SMTRes,...};` in following with the rest of this library's code?